Twig/LocalAssetTwigExtension.php

```diff
@@ -44,7 +47,7 @@ public function __construct(
     public function getFunctions()
     {
         return [
-            new \Twig_SimpleFunction('local_asset', [$this, 'getLocalAsset']),
+            new TwigFunction('local_asset', [$this, 'getLocalAsset']),
```

Comment:
@alexander-schranz do we need a bc bridge for these changes?

Reply:
just increase the twig version to `^1.35 || ^2.0 || ^3.0`